tests/test_dpo_trainer.py

```diff
@@ -472,7 +472,7 @@ def test_dpo_trainer_padding_token_is_none(self):
 
                 trainer.train()
 
-    def test_dpo_trainer_w_dataset_num_proc(self):
```

Comment:
why d you remove this test?

Reply:
forgive me I was testing some stuff and forgot to add it back I push the update changes just now